### ISSUE TITLE
`git reset HEAD~` deletes recorded time

### ISSUE THREAD

**Issue by diego-betto** [git-time-metric/gtm#89]
I've noticed that if you use `git reset HEAD~` to undo a commit will result in a lost of gtm entries.

Any clue?

**Comment by mschenk42** [comment]
That is correct.  When you commit, gtm calculates the commit time (via a post-commit hook) and files based on what is in the .gtm directory in the root of the project and then removes the processed editor events.  The time is then stored in a git note associated with the commit.  If you reset HEAD~1 it doesn't have anyway to restore the original editor events that are used to calculate the time.
